lisp_ease/src/queries/map-calibration.lisp

```diff
@@ -0,0 +1,34 @@
+; Contains useful functions which help to find out the offset between the OpenEase (VR) and the bullet world semantic map.
+; The scenario is, that an episode is recorded, in which the human places cereal objects on the edges
+; of the kitchen island table, and milk objects on the edges of the sink table, so that one can differentiate,
+; between the two tables, and can see how offset they are in the bullet world map.
+; Then the transform can be adjusted and the objects can be moved, until they are on the edges of the tables in the bullet world as well.
+
+
+; TODO: this doesn't quite work yet?? <<< check if it does
```

Comment:
Please add as a commentary or so, the current offset pose (or transformation) between VR world and bullet world.